components/brave_wallet_ui/components/desktop/network-filter-selector/index.tsx

```diff
@@ -38,13 +42,10 @@ export const NetworkFilterSelector = ({ networkListSubset }: Props) => {
   const [showNetworkFilter, setShowNetworkFilter] = React.useState<boolean>(false)
 
   // redux
-  const {
-    selectedNetworkFilter,
-    networkList: reduxNetworkList,
-    isTestNetworksEnabled
-  } = useSelector(({ wallet }: { wallet: WalletState }) => wallet)
-
   const dispatch = useDispatch()
+  const selectedNetworkFilter = useSelector(({ wallet }: { wallet: WalletState }) => wallet.selectedNetworkFilter)
+  const reduxNetworkList = useSelector(({ wallet }: { wallet: WalletState }) => wallet.networkList)
+  const isTestNetworksEnabled = useSelector(({ wallet }: { wallet: WalletState }) => wallet.isTestNetworksEnabled)
```

Comment:
Any particular reason for this? Also if we want to adopt this as a convention, makes sense to do a `useWalletSelector` (separate PR).

Reply:
https://react-redux.js.org/api/hooks#equality-comparisons-and-updates

"With useSelector(), returning a new object every time will always force a re-render by default. If you want to retrieve multiple values from the store, you can:

Call useSelector() multiple times, with each call returning a single field value"